### ISSUE TITLE
[CAIA A11y Research]: IVC Forms, CHAMPVA 10-10d (April & May 2024)

### ISSUE THREAD

**Issue by StevenStraily** [department-of-veterans-affairs/va.gov-team#80754]
> [!NOTE]
> **High-Level Overview**: IVD Forms | CHAMPVA
> - **Initial Kickoff**: `None`
> - **Research Timeframe**: `April 15th - May 2nd` _(A11Y 4/26-5/3/2024 +5/10/2024)_
> - **Time of Day:** `Morning`
> - **Pilot with a11ys** (_recommended_):  `NA - DUE TO TIMEFRAME`
> - **Assistive Tech Users**: `4` AT Users [(review the testing schedule table in this ticket)](https://github.com/department-of-veterans-affairs/va.gov-team/issues/80754#:~:text=and%20Skill%20level%3A-,Toggle%20to%20view%20the%20AT%20testing%20schedule,-P%23)
> - **Artifacts Available**: `Yes` [access them in the supporting artifacts section](https://github.com/department-of-veterans-affairs/va.gov-team/issues/80754#:~:text=Toggle%20to%20view%20the%20supporting%20artifacts)
> - **Testing Environment:** [Prototype in Staging](https://staging.va.gov/health-care/champva/apply-form-10-10d/)
> - **Assistive Tech Being Used**: `Screen Readers (3)` and `Screen Magnifier (1)` **_Types of AT - TBD_**
> - **Task Lists**: [review this ticket's task lists aka next steps/definition of done](https://github.com/department-of-veterans-affairs/va.gov-team/issues/80754#:~:text=Next%20Steps%3A%20Definition%20of%20Done)

## CAIA Assignee(s)

<details><summary>Toggle to view information about the CAIA Team.</summary>

- @coforma-jamie 
- @SarahKay8 
- @sara-amanda 

</details>

## About the Team

<details><summary>Toggle to view information about the team.</summary>

- **Team name:** IVC Forms
- **OCTO-DE product owner:** Premal Shah
- **Product manager:** Bo Altes
- **Designer**: Rachael Penfil, Steven Straily, Syd Hoeper, Jamie Fiore
- **FE engineer:** Michael Clement
- **Accessibility Champion(s):** Rachael Penfil (and future champion Steven Straily)
- **Product/team Slack channel:** [#ivc-forms-ux](https://dsva.slack.com/archives/C068Q9DQACX)
- **Ticket created by:** @StevenStraily 

</details>

## Research Sessions

> [!TIP]
> - Providing CAIA A11ys with the time slots in advance for AT users, will allow a11ys to avoid conflicts on the CAIA A11y Capacity Calendar, and reach out to other a11ys, should CAIA not be available. 
> - Providing CAIA A11ys with the technology and devices the users will be testing with, will allow CAIA to prepare in advance for the sessions. 

<details><summary>Toggle to view research timing information.</summary>

### Timeframe

- **Pilot Session** (recommended):`TBD`
- **Timeframe**: `April 15th - May 2nd` (A11Y starting 4/26/2024 - 5/3/2024 + 5/10/2024)

### Length of sessions

- **Session length:** `90` minutes
- **Buffer time between sessions:** `30 - 60` minutes
- **Maximum sessions per day:** `3`

</details>

###  Testing Schedule

CAIA A11Ys will be looking for device and AT information. We realize that some of this information will not be available until the day of the session. 

Please toggle and use the following table to enter the following information for each participant: Date and Time, Assistive tech (AT), Device and Operating System, Browser, and Skill level:

<details><summary>Toggle to view the AT testing schedule</summary>

P# | Date and Time (ET) | AT | Device and OS | Browser | Skill | CAIA A11ys Available
------------------|------------------|--------------|--------------|--------------|--------------|--------------
P14 | 4/26/1024, 9 a.m. EST (8 a.m. CST) - `RESCHEDULING - NO SHOW` | **Screen Reader** _Specific AT TBD_ | **Desktop/Laptop** _Specific OS TBD_ | _Specific Browser TBD_ | I**ntermediate?** uses every time they use the device | @SarahKay8 
P17 | 4/30/1024, 9 a.m. EST (8 a.m. CST) - `PARTICIPANT CANCELLED` | **Screen Reader** _Specific AT TBD_ | **Desktop/Laptop** _Specific OS TBD_ | _Specific Browser TBD_ | **Novice?** uses a few times a week | @coforma-jamie `Hard Stop`
P20 | 5/2/1024, 11 a.m. EST (10 a.m. CST) `TECHNICAL ISSUES, STOPPED MIDWAY`| **Screen Magnifier** | iOS, iPhone | Safari | Novice | @coforma-jamie 
P26 `CAIA left midway; participant only used captions` | 5/10/1024, 2:30 p.m. EST (1:30 a.m. CST) | Live captions | Desktop; Windows | Chrome | Advanced | @SarahKay8  

- **Consideration**: You can also consider asking participants more specific AT and device combinations. 
- **For instance:** TalkBack on Samsung Galaxy 8, or VoiceOver on iPad with magnification.

</details>

`COMPLETED` CAIA A11ys will review the coverage needed, and self-assign, based on a11y availability.

## Supporting artifacts 

<details><summary>Toggle to view the supporting artifacts</summary>

- [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/champva/1010D/product-outline.md)
- [User flows](https://www.figma.com/file/UmAtr3ULQEInMXfNFwP0g0/10-10d-User-Testing-Prep?type=design&node-id=1128-55365&mode=design&t=Zm9G4ic2ASxKyI1u-0)
- [Prototype or mockups](https://staging.va.gov/health-care/champva/apply-form-10-10d/)
- [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/champva/1010D/research/users/Usability-Accessibility%20Test%20(April)/Research%20Plan.md)
- [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/champva/1010D/research/users/Usability-Accessibility%20Test%20(April)/Conversation%20Guide.md)
- [Epics/issue that may be helpful](Insert URL here between the parenthesis)

</details>

## Next Steps: Definition of Done

```[tasklist]
### CAIA A11Y Tasks
- [x] **INTAKE TICKET:** @sara-amanda to process intake ticket
- [x] **INITIAL MEETING:** Schedule meeting with team **(Comms Via Slack)**
- [x] **(OPTIONAL) FEEDBACK REQUESTED:** Review Conversation Guide **NA - LIMITED TIMING**
- [x] **PILOT SESSION: SCHEDULE** Pilot session with the team. `Recommended` - **NA - LIMITED TIMING**
- [x] **PILOT SESSION: CREATE TICKET** Link ticket in this task list. **NA - LIMITED TIMING**
- [x] **CALENDAR HOLDS** Update A11y [CAIA Capacity Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/Chicago) with holds for user research sessions with AT, when dates become available.
- [x] **REMOVE CALENDAR HOLDS** Remove holds on A11y [**CAIA Capacity Calendar**](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/Chicago) and replace with invitations from project team (or Perigean) - `INVITES REQ. AND SENT C/O STEVEN 4/25` `HOLDS REMOVED 4/25`
- [x] **Divvy user research session support with CAIA A11ys** - based on availability and technology available to support.
- [x] **Review research folder and assets** @coforma-jamie @SarahKay8
- [x] **Participate in A11Y research**  **Date Span:**`4/26/2024` and `5/3/2024`
- [x] **SESSION NOTES**: Create CAIA notes document for a11ys to use during the sessions [IVC Forms CHAMPVA Notes Google Doc](https://docs.google.com/document/d/1_0Z8XcHxqOwupg_nru3bZSHuZV48BHeP7R70Pgg4tXo/edit?usp=sharing)
- [x] **SYNC:** Connect with CAIA A11ys on research findings (meeting)
- [ ] **Create/update the CAIA A11y Research Deliverable** This is dependent on the number of study participants CAIA supports. If minimal, A11Ys may share feedback only in the feedback Mural or this ticket. `n/a`
- [ ] **Share the completed deliverable** with the project team via Slack `n/a`
- [x] New session added c/o Steven Straily on 5/9 for 5/10 P26
```

```[tasklist]
### IVC Forms/CHAMPVA TEAM Tasks
- [x] Tag @coforma-terry in Slack [#sitewide-content-accessibility-ia](https://dsva.slack.com/archives/C01K37HRUAH) :thread:[ in DSVA Slack from Steven in Accessibility Help Channel 4/15/2024](https://dsva.slack.com/archives/C8E985R32/p1713215220775169)
- [x] Schedule intial meeting with @coforma-terry and @sara-amanda (SLACK COMMS)
- [x] Schedule a pilot session with CAIA a11ys `Recommended` **NA - LIMITED TIMING**
- [x] (OPTIONAL) Receive feedback on Conversation Guide for Screen Readers (if requested) **NA - LIMITED TIMING**
- [x] **PROTOTYPE:** Provide prototype to CAIA A11ys via this ticket in the supporting artifacts section [Prototype in Staging](https://staging.va.gov/health-care/champva/apply-form-10-10d/)
- [x] **DATES NEEDED** See `TESTING SCHEDULE SECTION` [Toggle to view the AT testing schedule](https://github.com/department-of-veterans-affairs/va.gov-team/issues/80754#:~:text=Toggle%20to%20view%20the%20AT%20testing%20schedule) in the ticket above. Please provide date range and slots for AT sessions being planned.
- [ ] **ASSISTIVE TECH. INFO NEEDED:** Please provide the AT the users will be utilizing during the research sessions, when known. See `TESTING SCHEDULE SECTION` [Toggle to view the AT testing schedule](https://github.com/department-of-veterans-affairs/va.gov-team/issues/80754#:~:text=Toggle%20to%20view%20the%20AT%20testing%20schedule) in the ticket above. **PARTIAL RCVD 4/25/2024; REMAINING INFO DURING SESSION**
- [x] **INVITES:** Forward invites from Perigean to CAIA A11ys (Sara or Terry) - 4/25/2024 INVITES REQ. AND FWD FROM STEVEN S.
- [x] New session added 5/9 for 5/10 via Steven Straily
```

**Comment by sara-amanda** [comment]
## CAIA Research Support 

Calendar holds have been placed for three sessions, and will be replaced with official invites, when those are received. 

## Initial Holds
- April 26th 8am CST [9 a.m. ET] - SK
- April 30th 8am CST [9 a.m. ET] - Jamie
- May 2nd 10am CST [11 a.m. ET] - Jamie

Ref. Slack: [Thread in DSVA](https://dsva.slack.com/archives/C8E985R32/p1713215220775169)

### Details Pending
- **Session Count:** 3 sessions known
- **Devices:** mobile or desktop
- **Assistive Tech:** AT not asked in screener

## Next Steps
Ticket Refinement by CAIA

**Comment by sara-amanda** [comment]
## Participant Information
c/o @StevenStraily via Slack. I will add this to the table in this ticket.
> P14 - 26th is on a Desktop/Laptop - and uses a screen reader every time they use their device
> P17 - 30th is on Desktop/Laptop - Screen reader a few times a week
> P20 - 2nd is on Mobile - Uses a Screen Magnifier a few times a month
